<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Llegamos aqui... desde el inicio de sesion</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bttnCelda1_1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="bttnCelda1_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="bttnCelda1_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>120</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="bttnCelda1_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="bttnCelda2_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="bttnCelda2_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="bttnCelda2_1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="bttnCelda2_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>170</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="bttnCelda4_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="bttnCelda3_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="bttnCelda4_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>270</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="bttnCelda3_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>220</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="bttnCelda3_1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="bttnCelda4_1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="bttnCelda4_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>270</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="bttnCelda3_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>220</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
